agent/api/container/container_test.go

```diff
@@ -479,3 +479,10 @@ func TestPerContainerTimeouts(t *testing.T) {
 	assert.Equal(t, container.GetStartTimeout(), expectedTimeout)
 	assert.Equal(t, container.GetStopTimeout(), expectedTimeout)
 }
+
+func TestSetRuntimeIdInContainer(t *testing.T) {
+	container := Container{
+		RuntimeId: "asdfghjkl1234",
+	}
+	assert.Equal(t, "asdfghjkl1234", container.GetRuntimeId())
```

Comment:
currently no test coverage on SetRuntimeId. Add a call to SetRuntimeId() to change the value then validate with another assert.